logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MemberMapper.kt

```diff
@@ -12,6 +12,7 @@ interface MemberMapper {
     fun fromMapOfClientsResponseToRecipients(qualifiedMap: Map<UserId, List<SimpleClientResponse>>): List<Recipient>
     fun fromApiModelToDaoModel(conversationMembersResponse: ConversationMembersResponse): List<PersistedMember>
     fun fromEventToDaoModel(members: List<ConversationMember>): List<PersistedMember>
+    fun fromDaoModel(entity: PersistedMember): Member
```

Comment:
Here I think we can simplify the naming by removing the `Model` from functions' names.

Reply:
I share the same feeling, but I tried to keep it consistent with the current functions.

I suggest renaming this in another PR and doing it for all at once.

Reply:
Totally agree! Thanks!